projects/1.20/assets/more-bows-cj/morebows/lang/zh_cn.json

```
@@ -0,0 +1,40 @@
+{
+	"item.morebows.copper_bow": "铜弓",
+	"item.morebows.iron_bow": "铁弓",
+	"item.morebows.golden_bow": "金弓",
+	"item.morebows.emerald_bow": "绿宝石弓",
+	"item.morebows.diamond_bow": "钻石弓",
+	"item.morebows.netherite_bow": "下界合金弓",
+	"item.morebows.gilded_netherite_bow": "镶金的下界合金弓",
+	"item.morebows.enderite_bow": "末影合金弓",
+	"item.morebows.gilded_enderite_bow": "镶金的末影合金弓",
+
+	"upgrade.morebows.bow_upgrade": "升级弓",
+	"item.morebows.smithing_template.bow_upgrade.additions_slot_description": "在此放入下一品质的方块",
+	"item.morebows.smithing_template.bow_upgrade.applies_to": "弓",
+	"item.morebows.smithing_template.bow_upgrade.base_slot_description": "在此放入需要升级的弓",
+	"item.morebows.smithing_template.bow_upgrade.ingredients": "铜, 铁, 金, 绿宝石, 钻石 块",
```

```suggestion
	"item.morebows.smithing_template.bow_upgrade.ingredients": "铜、铁、金、绿宝石、钻石 块",
```
应使用全角标点，此类问题请全文通查
并列的词汇之间应使用顿号而非逗号
这条在游戏内的具体显示情况？感觉单拿出一个“块”放在句尾是不是不太妥